src/compiler/crystal/syntax/parser.cr

```diff
@@ -103,7 +103,7 @@ module Crystal
         break if end_token?
       end
 
-      Expressions.from(exps)
+      Expressions.from(exps).at(exps.first)
```

Comment:
Note that I reproduce the CI failure locally (Ubuntu 14.04, LLVM 3.9 apt repository).